pkg/intermediate/aggregate.go

```diff
@@ -408,17 +408,10 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 			waitForReadyToSendRetries: 0,
 			isIPv4:                    isIPv4,
 		}
-
-		if !correlationRequired {
-			aggregationRecord.ReadyToSend = true
-			// If no correlation is required for an Inter-Node record, K8s metadata is
-			// expected to be not completely filled. For Intra-Node flows and ToExternal
-			// flows, areCorrelatedFieldsFilled is set to true by default.
-			if flowType == registry.FlowTypeInterNode {
-				aggregationRecord.areCorrelatedFieldsFilled = false
-			} else {
-				aggregationRecord.areCorrelatedFieldsFilled = true
-			}
+		// For intra-node and external traffic, areCorrelatedFieldsFilled is always true.
+		// For inter-node with allow np/anp action, its areExternalFieldsFilled will be set to true once the correlation job is finished.
+		if flowType != registry.FlowTypeInterNode {
+			aggregationRecord.areCorrelatedFieldsFilled = true
```

Comment:
It seems like `areCorrelatedFieldsFilled` is only being used in UT? If that's the case, do we still want to keep it? maybe I missed something

Reply:
It will be set to true in line#355. And areCorrelatedFieldsFilled will be used in the flowaggregator.go in Antrea repo as well.